### DISCUSSION TITLE
halo2/chap-1/index

### DISCUSSION THREAD

**Discussion by giscus** [z2o-k7e/z2o-k7e.github.io#58] [Announcements]
# halo2/chap-1/index

零知世界, Halo2 Tutorial, world of Zero-knowledge proof, Halo2 教程

https://learn.z2o-k7e.world/halo2/chap-1/index.html

**Comment by SylvanasW1ndrunner** [comment]
这里c不也是private input吗，为什么要用load constant？
let circuit = MyCircuit {
            c,
            a: Value::known(a),
            b: Value::known(b),
        };
这里为什么a,b是known呢？
